src/init.cpp

```diff
@@ -456,7 +456,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-printtoconsole", _("Send trace/debug info to console instead of debug.log file"));
     if (showDebug)
     {
-        strUsage += HelpMessageOpt("-printpriority", strprintf("Log transaction priority and fee per kB when mining blocks (default: %u)", DEFAULT_PRINTPRIORITY));
+        strUsage += HelpMessageOpt("-printpriority", strprintf("Log transaction fee per kB when mining blocks (default: %u)", DEFAULT_PRINTPRIORITY));
```

Comment:
Agree with @sdaftuar . Commit at https://github.com/jnewbery/bitcoin/commit/b83daf977f38d98a3b3aa9ed5857bfd2f19b2a09 adds `-printfee` option. Feel free to pull into this PR if you want it, or I'll open another PR after this gets merged.